x/tokenomics/keeper/keeper_test.go

```diff
@@ -111,9 +117,9 @@ func (s *TestSuite) TestClaimSettlement_ClaimPendingBeforeSettlement() {
 
 	// 2. Settle pending claims just after the session ended.
 	// Expectations: Claims should not be settled because the proof window hasn't closed yet.
-	// TODO_BLOCKER(@red-0ne): Use the governance parameters for more
-	// precise block heights once they are implemented.
-	blockHeight = claim.SessionHeader.SessionEndBlockHeight + 2 // session ended but proof window is still open
+	blockHeight = shared.GetProofWindowOpenHeight(
+		&sharedParams, claim.SessionHeader.SessionEndBlockHeight,
+	) + 2 // session ended but proof window is still open
```

Comment:
Can we make +2 a function of parameters?

Reply:
Are you suggesting passing `claim.SessionHeader.SessionEndBlockHeight + 2` to the `shared.GetProofWindowOpenHeight()` call? 

That wouldn't have any effect because the second argument (`queryHeight`) will map to the same session and therefore to the same proof window open height.

Reply:
```go
proofWindowOpenHeight := ...
proofWindowCloseHeight := 

# Set block height to sometime between the proof window height opening and closing
blockHeight = proofWindowOpenHeight + (proofWindowCloseHeight - proofWindowOpenHeight) / 2
```

Reply:
Thanks for clarifying, I understand what you meant now. :+1: